frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -189,14 +189,19 @@ const numberOfIncompleteReceivers = (): number => {
   const { route, receivers } = config;
   const { receiver: defaultReceiverName } = route;
 
+  // if no receivers or default receiver, then no longer initial setup, hide info alerts
+  if (!receivers || !defaultReceiverName) {
+    return -1;
```

Comment:
It's weird for a function called `numberOfIncompleteReceivers` to ever return `-1`.

I'm not sure I see where we handle the `-1` return value.

Reply:
[here](https://github.com/openshift/console/blob/a1a44f9df5c4fb85e2f4f007ccabffde419667db/frontend/public/components/monitoring/alert-manager-config.tsx#L393), returning `-1` to indicate a non-greenfield setup and to not show infotips.

Reply:
> [here](https://github.com/openshift/console/blob/a1a44f9df5c4fb85e2f4f007ccabffde419667db/frontend/public/components/monitoring/alert-manager-config.tsx#L393), returning `-1` to indicate a non-greenfield setup and to not show infotips.

Why not return 0 here in that case?

Reply:
Thought `-1` was clearer, usually denotes error case, which not being greenfield seemed to indicate.